resources/logfs/dl.css

```diff
@@ -54,4 +69,7 @@ header {
   width: 167px;
   height: 40px;
   padding: 10px 15px;
-}
\ No newline at end of file
+}
+#v {
+  display: none;
+}
```

Comment:
Even if we back away from this PR we should definitely have this. No idea why I removed it in JavaScript on #126.